scout/server/blueprints/variant/templates/variant/buttons.html

```diff
@@ -124,12 +124,14 @@
   <form role="form" action="{{ url_for('alignviewers.igv') }}" target="_blank" method="POST">
```

Comment:
Ah, right; I had forgotten about that one! I could make an attempt; it would indeed touch about the same code! But maybe with this working we could also merge and have it available while refactoring.

Reply:
> Another thing unrelated to this PR, but that gets obvious by testing it, is that **the IGV track is super slow to load in general**. It becomes even slower when you attempt at loading it for the whole genome view, and the thing is that we don't even need to display it if we are zoomed out so much! We should think about a solution (in another PR!)

Its could "just" be a matter of setting appropriate visibilityWindow? Which track do you mean; the ClinVar ones?

Reply:
> Its could "just" be a matter of setting appropriate visibilityWindow? Which track do you mean; the ClinVar ones?

Yes exactly, I meant to write clinvar but I wrote igv. It's probably the visibility window, yes!